src/vm/excep.cpp

```diff
@@ -7164,6 +7164,13 @@ EXTERN_C void JIT_MemCpy_End();
 
 EXTERN_C void JIT_WriteBarrier_End();
 EXTERN_C void JIT_CheckedWriteBarrier_End();
+#else
+#ifdef FEATURE_PAL
+EXTERN_C void JIT_WriteBarrierGroup();
```

Comment:
Thank you you liked it. `JIT_WriteBarrierStart` and `JIT_WriteBarrierLast` is used in jitinterfacex86.cpp; so they seem to be needed.
```
    // All write barrier helpers should fit into one page.
    // If you hit this assert on retail build, there is most likely problem with BBT script.
    _ASSERTE_ALL_BUILDS("clr/src/VM/i386/JITinterfaceX86.cpp", (BYTE*)JIT_WriteBarrierLast - (BYTE*)JIT_WriteBarrierStart < PAGE_SIZE);
    _ASSERTE_ALL_BUILDS("clr/src/VM/i386/JITinterfaceX86.cpp", (BYTE*)JIT_PatchedWriteBarrierLast - (BYTE*)JIT_PatchedWriteBarrierStart < PAGE_SIZE);
```